includes/controllers/class.llms.controller.orders.php

```diff
@@ -129,9 +129,9 @@ public function confirm_pending_order() {
 		 *
 		 * @since 3.34.4
 		 *
-		 * @param bool $can_be_confirmed True if the order can be confirmed, false otherwise.
-		 * @param LLMS_Order $order Order object.
-		 * @param string $gateway_id Payment gateway ID.
+		 * @param bool       $can_be_confirmed   True if the order can be confirmed, false otherwise.
+		 * @param LLMS_Order $order              Order object.
+		 * @param string     $gateway_id Payment Gateway ID.
```

Comment:
```suggestion
		 * @param string     $gateway_id Payment gateway ID.
```
retriggering build with something that doesn't matter